project.clj

```diff
@@ -24,7 +24,7 @@
                  [org.cyverse/authy "2.8.0"]
                  [org.cyverse/clojure-commons "2.8.0"]
                  [org.cyverse/kameleon "2.8.0"]
-                 [org.cyverse/mescal "2.8.0"]
+                 [org.cyverse/mescal "2.8.1-SNAPSHOT"]
```

Comment:
Ah, yeah, sorry, I didn't notice that mescal was already at `2.8.1-SNAPSHOT`.
I was thinking it was still at `2.8.0`.